astroquery/mast/core.py

```diff
@@ -215,14 +215,18 @@ def service_request_async(self, service, params, pagesize=None, page=None):
             The Mashup service to query.
         params : dict
             JSON object containing service parameters.
-        pagesize : int or None, optional
+        pagesize : int, optional
             Default None.
             Can be used to override the default pagesize (set in configs) for this query only.
             E.g. when using a slow internet connection.
-        page : int or None, optional
+        page : int, optional
             Default None.
             Can be used to override the default behavior of all results being returned to obtain
             a specific page of results.
+        **kwargs : 
+            See MashupRequest properties 
+            `here <https://mast.stsci.edu/api/v0/class_mashup_1_1_mashup_request.html>`_ 
```

Comment:
you're reusing the ``here`` reference in all your links and that's causing the sphinx build to fail. For one-off linking add double underscore to the end everywhere, rather than a singe.